README.rst

```diff
@@ -1,7 +1,10 @@
 CrateDB Docs Theme
 ==================
 
-A Sphinx_ theme for the `CrateDB documentation`_.
+A Sphinx_ theme for the `CrateDB documentation`_. This theme can be used as an 
```

Comment:
```suggestion
A Sphinx_ theme for the `CrateDB documentation`_.

*Note: this theme can be used as an 
```

(this will need rewrapping after breaking it apart)